apps/generic-issuance-client/src/components/RefreshSession.tsx

```diff
@@ -8,9 +8,18 @@ export const RefreshSession: FC = () => {
   useEffect(() => {
     const refresh = (): void => {
       if (stytchClient.session.getSync()) {
-        stytchClient.session.authenticate({
-          session_duration_minutes: SESSION_DURATION_MINUTES
-        });
+        const expiresAt = stytchClient.session.getSync()?.expires_at;
+        if (expiresAt != null) {
+          const expirationDate = new Date(expiresAt);
+          if (
+            Date.now() >
+            expirationDate.getTime() - SESSION_DURATION_MS * 1.3
```

Comment:
where does the 1.3 come from?

Reply:
i just reverted changes to this file to avoid answering this question XD